### PR TITLE
Closes #503

### LINKED ISSUE #503: add plugin tools line item to main menu
similar to `core tools:`

### PR DESCRIPTION
**Pull request by bpagon13**
I created a method in meta.py called Tools_Status that can also do the same thing as Core() I believe. I wanted to consolidate the two (plugins and core) into one method because I believed the searching for them was pretty similar. I kept Core() around though in case there was something wrong with Tools_Status or if you didn't think consolidating them is the right move.

### CODE REVIEW

File: vent/helpers/meta.py [hunk 1]
```diff
@@ -285,6 +285,95 @@ def Core(branch="master", **kargs):
...
+    Get tools that are currently installed/built/running and also the number of
+    repos that those tools come from; can toggle whether looking for core tools
+    or plugin tools
+    """
+    # !! TODO this might need to store namespaces/branches/versions
+    all_tools = {'built': [], 'running': [], 'installed': [], 'normal': []}
+    core_repo = 'https://github.com/cyberreboot/vent'
+    repos = set()
+
+    # get list of tools
```
Comment (cglewis) [inline vent/helpers/meta.py]: there's already a helper function that does this `Tools`

**Comment by cglewis** [comment]
@bpagon13 looks good, go ahead and consolidate them down to just one function, you can remove `Core`.  nice job.

File: vent/api/actions.py [hunk 2]
```diff
@@ -603,7 +603,7 @@ def cores(self, action, branch="master"):
         status = (True, None)
         try:
             self.logger.info("action provided: " + str(action))
-            core = Core(branch=branch)
+            core = Tools_Status(True, branch=branch)
```
Comment (cglewis) [inline vent/api/actions.py]: this has 2 return values inside the tuple now, correct?

File: vent/api/actions.py [hunk 3]
```diff
@@ -11,7 +11,7 @@
 from vent.api.templates import Template
 from vent.helpers.logs import Logger
 from vent.helpers.meta import Containers
-from vent.helpers.meta import Core
+from vent.helpers.meta import Tools_Status
```
Comment (cglewis) [inline vent/api/actions.py]: alphabetize, please.

File: tests/test_helpers_meta.py [hunk 4]
```diff
@@ -2,7 +2,7 @@
 import os
 
 from vent.helpers.meta import Containers
-from vent.helpers.meta import Core
+from vent.helpers.meta import Tools_Status
```
Comment (cglewis) [inline tests/test_helpers_meta.py]: alphabetize, please.

File: vent/menus/main.py [hunk 5]
```diff
@@ -10,7 +10,7 @@
 
 from vent.api.actions import Action
 from vent.helpers.meta import Containers
-from vent.helpers.meta import Core
+from vent.helpers.meta import Tools_Status
```
Comment (cglewis) [inline vent/menus/main.py]: alphabetize, please.

File: vent/api/actions.py (hunk 2 above)
Comment (bpagon13) [inline vent/api/actions.py]: Yes, you are correct. I missed that, I'll clean it up now

**Pull request merged**